### PR TITLE
This fixes all pylint warnings and adds it to the tox run

### PR DESCRIPTION
**Pull request by plars**

### CODE REVIEW

**Comment by jocave** [comment]
I hit a corner case! If you have run snapcraft in the source directory and not cleaned up afterwards, the tests seem to start running against everything in e.g. the stage, prime directories etc. Is this something we could/should try to handle?

**Comment by plars** [comment]
I'm having some trouble reproducing that, perhaps because I don't build with --destructive-mode?  I'd be curious if this one line change would fix it for you. If so, I'd be happy to add this, but prefer to do it on a separate PR:
```
diff --git a/tox.ini b/tox.ini
index e51593c..04b192a 100644
--- a/tox.ini
+++ b/tox.ini
@@ -17,4 +17,4 @@ commands =
     {envbindir}/python setup.py develop
     {envbindir}/python -m flake8 setup.py testflinger_cli
     {envbindir}/python -m pylint testflinger_cli
-    {envbindir}/python -m pytest --doctest-modules --cov=.
+    {envbindir}/python -m pytest --doctest-modules --cov=testflinger_cli
```

**Comment by plars** [comment]
after discussion on mm, merging this as-is
